### DISCUSSION TITLE
how to validate input after user finished typing?

### DISCUSSION THREAD

**Discussion by nsmichelj** [ant-design/ant-design#41318] [General]
I want to create an input that when the user finishes entering the respective data, a request is made to the backend to verify if the entered data already exists, just as it does, for example, gmail with the name of the email, if it already exists then what turns red and the error message is displayed.  And tried to use lodash's debounce function and even my own hook to execute the request after 2 seconds, however I didn't manage to succeed in the validation, does anyone know how to do this, I would be very grateful.

**Comment by Yuiai01** [comment]
Hello, you can use the custom validator of `Form`，You can call the back-end check method in the custom validator and then return a different check result based on the result returned by the back-end。You can look at my example https://codesandbox.io/s/ji-ben-shi-yong-antd-5-3-1-forked-i0k70m?from-embed=&file=/demo.tsx. I hope I can help you.
